### DISCUSSION TITLE
使用java 的 org.apache.poi的包操作ppt转为json或者xml

### DISCUSSION THREAD

**Discussion by Hellofffff** [pipipi-pikachu/PPTist#102] [Ideas]
后端可以org.apache.poi的包操作ppt并转为json或者xml，还原度非常高，非常复杂的ppt也能还原，目前我没找到不能还原的， 然后通过json转换为html幻灯片

**Comment by BOKE2014** [comment]
Will try. 3Q

**Comment by TheBiiigBlue** [comment]
您好，您有相关的代码可以供学习下吗，我在转换过程中遇到很多问题，比如比例的转换、前端的一些属性不知道从哪里提取、前端的json怎么转成pptx的xml等，非常头疼

**Comment by jrs64** [comment]
@Hellofffff 您好，请问一下ppt中动画是如何解析的
